iocore/hostdb/I_HostDBProcessor.h

```diff
@@ -139,10 +138,49 @@ struct SRVInfo {
   unsigned int key;
 };
 
-struct HostDBInfo {
+struct HostDBInfo: public RefCountObj {
   /** Internal IP address data.
       This is at least large enough to hold an IPv6 address.
   */
+
+  int iobuffer_index;
+  static HostDBInfo* alloc(int size=0) {
+    size += sizeof(HostDBInfo);
+    int iobuffer_index = iobuffer_size_to_index(size);
```

Comment:
The size varies quite a bit based on size of hostname, and number of records. In my local testing of sizing its usually in the hundreds of bytes (simple single record is ~100 bytes -- and an RR with 2 responses is ~260 bytes)